language/transaction-builder/generator/tests/generation.rs

```diff
@@ -298,6 +298,83 @@ fn test_that_java_code_compiles_and_demo_runs() {
     assert!(output.status.success());
 }
 
+#[test]
+#[ignore]
+fn test_that_csharp_code_compiles_and_demo_runs() {
+    let registry = get_diem_registry();
+    let abis = get_stdlib_script_abis();
+    let dir = tempdir().unwrap();
+
+    let paths = std::fs::read_dir("examples/csharp/custom_diem_code")
+        .unwrap()
+        .map(|e| e.unwrap().path());
+    let config = serdegen::CodeGeneratorConfig::new("Diem.Types".to_string())
+        .with_encodings(vec![serdegen::Encoding::Bcs])
+        .with_custom_code(buildgen::read_custom_code_from_paths(
+            &["Diem", "Types"],
+            paths,
+        ));
+    let bcs_installer = serdegen::java::Installer::new(dir.path().to_path_buf());
+    bcs_installer.install_module(&config, &registry).unwrap();
+    bcs_installer.install_serde_runtime().unwrap();
+    bcs_installer.install_bcs_runtime().unwrap();
+
+    let abi_installer = buildgen::java::Installer::new(dir.path().to_path_buf());
+    abi_installer
+        .install_transaction_builders("Diem.Stdlib", &abis)
+        .unwrap();
+
+    std::fs::copy(
+        "examples/csharp/StdlibDemo.cs",
+        dir.path().join("StdlibDemo.cs"),
+    )
+    .unwrap();
+
+    let paths = || {
+        std::iter::empty()
+            .chain(std::fs::read_dir(dir.path().join("Serde")).unwrap())
+            .chain(std::fs::read_dir(dir.path().join("BCS")).unwrap())
+            .chain(std::fs::read_dir(dir.path().join("Diem/Types")).unwrap())
+            .chain(std::fs::read_dir(dir.path().join("Diem/Stdlib")).unwrap())
+            .map(|e| e.unwrap().path())
+            .chain(std::iter::once(dir.path().join("StdlibDemo.java")))
+    };
+
+    let status = Command::new("javadoc")
```

Comment:
We can probably start without any support for documentation in C#.

Reply:
Yeah, I removed this and updated the entire test suite to handle C# only, no docs.